grpc-api/src/main/proto/geese.proto

```diff
@@ -47,6 +47,16 @@ message GeeseResponse {
     repeated GeeseLine lines = 1;
 }
 
+/*
+ * Geese Types
+ */
+enum GooseType {
+    CANADA_GOOSE = 0;
```

Comment:
By convention, enum value name should start with type: `GOOSE_TYPE_CANADA_GOOSE`

Reply:
Good point, will address.

Reply:
Fixed